playground/nextjs/pages/_app.tsx

```diff
@@ -39,6 +39,17 @@ export default function App({ Component, pageProps }: AppProps) {
             <Head>
                 <title>PostHog</title>
                 <meta name="viewport" content="width=device-width, initial-scale=1" />
+                {/* CSP - useful for testing our documented recommendations. NOTE: Unsafe is only needed for nextjs pre-loading */}
+                <meta
+                    http-equiv="Content-Security-Policy"
+                    content={`
+                    default-src 'self';
+                    connect-src 'self' http://localhost:8000 https://*.posthog.com;
```

Comment:
Minor nit but there's a script in package.json (called dev-crossdomain) which uses https (used for testing if some features work with https). It'd be great if this could use https instead if `NEXT_PUBLIC_CROSSDOMAIN==1`.

Reply:
Updated. If its not correct, feel free to fix forward 👍